pro/src/components/IndividualOfferForm/OfferLocation/OfferLocation.tsx

```diff
@@ -0,0 +1,132 @@
+import { useField, useFormikContext } from 'formik'
+import React, { useState } from 'react'
+
+import { VenueListItemResponseModel } from 'apiClient/v1'
+import { AddressSelect } from 'components/Address/Address'
+import { AddressManual } from 'components/AddressManual/AddressManual'
+import { FormLayout } from 'components/FormLayout/FormLayout'
+import fullBackIcon from 'icons/full-back.svg'
+import fullNextIcon from 'icons/full-next.svg'
+import { Button } from 'ui-kit/Button/Button'
+import { ButtonVariant } from 'ui-kit/Button/types'
+import { RadioButton } from 'ui-kit/form/RadioButton/RadioButton'
+import { TextInput } from 'ui-kit/form/TextInput/TextInput'
+
+import { IndividualOfferFormValues } from '../types'
+
+import { OFFER_LOCATION } from './constants'
+import styles from './OfferLocation.module.scss'
+
+export interface OfferLocationProps {
+  venue: VenueListItemResponseModel | undefined
+}
+
+export const OfferLocation = ({ venue }: OfferLocationProps): JSX.Element => {
+  const formik = useFormikContext<IndividualOfferFormValues>()
+
+  const [showOtherAddress, setShowOtherAddress] = useState(
+    formik.values.offerlocation === OFFER_LOCATION.OTHER_ADDRESS
+  )
+  const [manuallySetAddress, , { setValue: setManuallySetAddress }] =
+    useField('manuallySetAddress')
+
+  const onManuallySetAddress = async () => {
+    const newValue = !manuallySetAddress.value
```

Comment:
Détail, mais plus compréhensible, surtout avec le if
```suggestion
    const isAdressManual = !manuallySetAddress.value
```

Reply:
Done